testing/etckeeper/APKBUILD

```diff
@@ -0,0 +1,50 @@
+# Contributor: Henrik Riomar <henrik.riomar@gmail.com>
+
+pkgname=etckeeper
+pkgver=1.18.6
+pkgrel=1
+pkgdesc="Store /etc in git."
```

Comment:
I generally dislike this, because git tracks history, and in tmpfs installs it means the whole history is also stored in the apkovl. Which is part of the reason why we wrote `aaudit`. Then again, it's much less known, less feature complete etc. And I do understand that people are possibly familiar with `etckeeper` so I think we should include it as an option.